### DISCUSSION TITLE
Withdrawl

### DISCUSSION THREAD

**Discussion by cinta165** [aptos-labs/aptos-developer-discussions#569] [General]
I made a withdrawal
0xa2b5779c9732d11f232baa2a0bbed2745da661fea73b66517d8eefc656c5b2d7

The status shows as successful, but it is not in the wallet.
![Uploading Screenshot_20241221_225432_Bybit.jpg…]()

**Comment by jkjun4574** [comment]
#551

**Reply by jkjun4574** [reply]
@cinta165

**Comment by jkjun4574** [comment]
#572

**Comment by yoga704** [comment]
Hello sir, I have a problem with a withdrawal, the notification that it has been sent to the destination address is still not there
Here is the address where you can help
0x16b26d9e0cd5cae7dae654b82bfa922730e6cab93465fe9a2543a9675cb7d553
#551 
#572
